### PR TITLE
Adds location class. Issue #5

### PR DESCRIPTION
**Pull request by jamalhansen**
This pull request adds a location class to resolve issue #5

### CODE REVIEW

**Comment by jcode89** [comment]
Looking good!

**Comment by chris9361** [comment]
Agreed

File: adventure/location.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+class Location(object):
```
Comment (jamalhansen) [inline adventure/location.py:2]: I like this class, I would prefer that you add a docstring.  Thanks!

Comment (jamalhansen) [inline adventure/location.py:2]: Thanks!  I enjoy writing docstrings and talking to myself on github!  I made the change.

**Comment by jamalhansen** [comment]
Thank you

**Pull request closed without merging**